java/Cherepnin_Oleh/src/com/zoolatech/lecture2/tasks/_2/RobotController.java

```diff
@@ -0,0 +1,116 @@
+package com.zoolatech.lecture2.tasks._2;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class RobotController {
+    private static Scanner scanner = new Scanner(System.in);
+    private Robot robot;
+
+    public void controlRobot() {
+        boolean flag = true;
+        showInstruction();
+        while (flag) {
+            System.out.println("Input operation:");
+            String action = scanner.next();
+            switch (action.toUpperCase(Locale.ROOT)) {
+                case "M" -> {
+                    System.out.println("Input the number of steps:");
+                    robot.moveForward(scanner.nextInt());
+                }
+                case "L" -> robot.turnLeft();
+                case "R" -> robot.turnRight();
+                case "V" -> robot.showPosition();
+                case "E" -> flag = false;
+                case "I" -> showInstruction();
+                default -> System.out.println("Incorrect operation \nTo view instruction press: 'I'");
+            }
+        }
+    }
+
+    private void showInstruction() {
+        System.out.println("To move forward press: 'M'\n" +
+                "To turn left press: 'L'\n" +
+                "To turn right press: 'R'\n" +
+                "To view current position press: 'V'\n" +
+                "To stop program press: 'E'\n" +
+                "To view instruction press: 'I'");
+    }
+
+    public void createRobot() {
```

Comment:
Try to define methods in the logical order: I think `createRobot` comes before `controlRobot`